flink-runtime/src/main/java/org/apache/flink/runtime/operators/Driver.java

```diff
@@ -87,4 +87,6 @@
 	 * @throws Exception Exceptions may be forwarded.
 	 */
 	void cancel() throws Exception;
+
+	void resetForIterativeTasks() throws Exception;
```

Comment:
I saw that code. But am not sure how to do that. Driver is actually instantiated from the config. So how to ensure that it is of type ResettableDriver? You mean all drivers will now become a type of ResettableDriver?